components/private/Color/src/UIColor+MaterialDynamic.h

```diff
@@ -26,4 +26,14 @@
 + (nonnull UIColor *)colorWithUserInterfaceStyleDarkColor:(nonnull UIColor *)darkColor
                                              defaultColor:(nonnull UIColor *)defaultColor;
 
+/**
+ Returns the version of the current color that takes the specified traits into account.
+
+ @param traitCollection The traits to use when resolving the color information.
+ @return The version of the color to display for the specified traits. On pre-iOS 13 the original
+ color is returned.
```

Comment:
Please document the behavior of this API on pre-iOS 13 devices.

Reply:
Isn't that done in the second sentence?

Reply:
Ah! Apologies; I'm used to seeing the older iOS version behaviors called out explicitly in the body content.

Reply:
Do you think a `@note blah blah blah` would be better?

Reply:
It's a pretty distinct behavioral change on older iOS versions, so it may be helpful for docs-skimming-folk like myself to have a specific `@note` callout.

Reply:
Sounds good, I updated it. My original thought was to have it in a separate but talking about a _return_ value outside of _return_ didn't really make sense to me but I was on the fence. I think as someone skimming it can easily be missed so let's error on the side of caution and add it separate as you stated.